### DISCUSSION TITLE
Use of `class` deviates from standard React practice

### DISCUSSION THREAD

**Discussion by stevenvachon** [perspective-dev/perspective#1362] [General]
Per #865, `class` was defined instead of `className`. It's also causing issues with Enzyme's `hasClass` method.

**Comment by texodus** [comment]
Thanks for the report!

This is correct.  The root issue IMO is incomplete Web Component support in React itself, as I alluded to [here](https://github.com/finos/perspective/issues/865#issuecomment-579180785).  I'm unclear of how to resolve this without writing a custom React wrapper around `<perspective-viewer>`, which seems like a bit of overkill perhaps (suggestions are welcome!).  Let me do a bit of research on this and see if there is a pattern we can follow from other Web Component libraries, otherwise we'll look into publishing a `@finos/perspective-viewer-react`.
